js/libs/ui-shared/src/user-profile/UserProfileFields.tsx

```diff
@@ -199,19 +217,12 @@ function determineInputType(
 
   const inputType = attribute.annotations?.inputType;
 
-  // If the attribute has no valid input type, fall back to a default input type.
-  // Depending on the length of the value, we either use a 'multi-input' or a 'text' input type so all values are always visible.
+  // If the attribute has no valid input type, it is always multi-valued.
   if (!isValidInputType(inputType)) {
-    return Array.isArray(value) && value.length > 1 ? "multi-input" : "text";
```

Comment:
This fixes a bug, so that the multi and text values are handled correctly in the absence of an input type. This should probably be restored.

Reply:
right I have the same fallback just optimised it a bit